### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by juhoeerola**
Couple things I've come across recently. Each commit is hopefully self explanatory enough.

### CODE REVIEW

File: frontend/src/components/NavBar.module.scss [hunk 1]
```diff
@@ -49,12 +47,9 @@
...
   align-items: center;
 
-  padding-right: 10px;
   color: $COLOR_BLACK100;
-  margin-top: 12px;
-  margin-bottom: 12px;
-  padding-left: 6px; // Areas meet at the middle
-  margin-left: -6px; // Added - margin to match the padding
+  margin: 12px 0;
+  padding: 0 6px;
```
Comment (IidaKainu) [inline frontend/src/components/NavBar.module.scss:52]: I'd remove the padding here (affects user and logout -icons) to follow the design better. Now there's quite a large gap between them
```suggestion
```

**Review by IidaKainu — CHANGES_REQUESTED** [review]
Great improvements! Found one change request though

File: frontend/src/components/NavBar.module.scss (hunk 1 above)
Comment (juhoeerola) [reply frontend/src/components/NavBar.module.scss]: agreed, removed now

**Comment by juhoeerola** [comment]
rebased on master

**Pull request merged**